lib/dbConnections/connection.js

```diff
@@ -0,0 +1,16 @@
+const pg = require('pg');
+const Promise = require("bluebird");
+const config = require('../constants/dbConfig.js');
+Promise.promisifyAll(pg.Client.prototype);
+Promise.promisifyAll(pg.Client);
+Promise.promisifyAll(pg.Connection.prototype);
+Promise.promisifyAll(pg.Connection);
+Promise.promisifyAll(pg.Query.prototype);
+Promise.promisifyAll(pg.Query);
+Promise.promisifyAll(pg, {
+    context: pg
+});
+
+const conString = `postgres://${config.dbUser}:${config.dbPassword}@${config.dbHost}:${config.dbPort}/${config.dbName}`;
+const client = new pg.Client(conString);
```

Comment:
So, you use the same PG client in all cases when this file is loaded. Please consider using a connection pool